utils/buildcli/README.md

```diff
@@ -0,0 +1,10 @@
+# builcli
+
+`buildcli` is a Python module containing functions and data structures used by
+the Swift build script.
+
+You may run unit tests for `buildcli` from the command line:
+
+```sh
+apple/swift $ python -m unittest discover -s utils/buildcli
```

Comment:
Ooh, great idea!! Will do. I'll also considering doing this for the tests I added to `utils/cmpcodesize`. Thanks for the tip!!